File: app/src/main/java/mariavv/fitnesspal/other/Const.java
package mariavv.fitnesspal.other;

import mariavv.fitnesspal.FitnessPal;

public class Const {

    public static final String LOG_TAG = FitnessPal.class.getSimpleName();

    static final String DATE_PATTERN_DEFOULT = "dd MMMM yyyy";

    public static class Screen {
        public static final String JOURNAL = "journal";
        public static final String HANDBOOK = "hand_book";
        public static final String ADD_DISH = "add_dish";
        public static final String ADD_FOOD = "add_food";
    }

    public static class BundleArg {
        public static final String ARG_DATE = "date";
    }
}

File: app/src/main/java/mariavv/fitnesspal/other/Const.kt
package mariavv.fitnesspal.other

import mariavv.fitnesspal.App

object Const {

    //нужен
    val LOG_TAG = App::class.java.getSimpleName()

    internal val DATE_PATTERN_DEFOULT = "dd MMMM yyyy"

    object Screen {
        val JOURNAL = "journal"
        val HANDBOOK = "hand_book"
        val ADD_DISH = "add_dish"
        val ADD_FOOD = "add_food"
    }

    object BundleArg {
        val ARG_DATE = "date"
    }
}
